### PR TITLE
Add Web Bluetooth API feature

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Following https://github.com/Rik/platform-status/pull/2, here's the Web Bluetooth API.

### CODE REVIEW

File: features/web_bluetooth.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Web Bluetooth API
+category: apps
+bugzilla: 1204396
+firefox_status: under consideration
```
Comment (marco-c) [inline features/web_bluetooth.md]: I'd say "in-development", since there's a clear indication that Firefox is going to implement it.

**Comment by beaufortfrancois** [comment]
Thank you @marco-c!
I've fixed it.

**Comment by marco-c** [comment]
Thank you!

**Comment by beaufortfrancois** [comment]
Quick question: Should I expect it to be live soon or do we have to wait a little bit?

**Pull request merged**